Add support webpack for work in browser

I have an issue 

ERROR in ./node_modules/pofile/lib/po.js
26 unchanged chunks
chunk {main} main.js, main.js.map (main) 2.91 MB [initial] [rendered]
Module not found: Error: Can't resolve 'fs' in '/Users/user/work/project/node_modules/pofile/lib'
chunk {vendor} vendor.js, vendor.js.map (vendor) 8.2 MB [initial] [rendered]
｢wdm｣: Failed to compile.

maybe you can create split builds for nodejs and browser?

--- a/lib/po.js
+++ b/lib/po.js
@@ -1,5 +1,3 @@
-var fs = require('fs');
-
 function trim(string) {
     return string.replace(/^\s+|\s+$/g, '');
 }
@@ -13,7 +11,7 @@ var PO = function () {
 };
 
 PO.prototype.save = function (filename, callback) {
-    fs.writeFile(filename, this.toString(), callback);
+    require('fs').writeFile(filename, this.toString(), callback);
 };
 
 PO.prototype.toString = function () {
@@ -65,7 +63,7 @@ PO.prototype.toString = function () {
 };
 
 PO.load = function (filename, callback) {
-    fs.readFile(filename, 'utf-8', function (err, data) {
+    require('fs').readFile(filename, 'utf-8', function (err, data) {
         if (err) {
             return callback(err);
         }

